corehq/apps/app_manager/translations.py

```diff
@@ -569,11 +574,16 @@ def update_form_translations(sheet, rows, missing_cols, app):
     return msgs
 
 
-def _escape_output_value(value):
-    return etree.fromstring("<value>" +
-        re.sub("(?<!/)>", "&gt;", re.sub("<\s*(?!output)", "&lt;", value)) +
-        "</value>")
-
+def escape_output_value(value):
+    try:
+        return etree.fromstring(u"<value>{}</value>".format(
+            re.sub("(?<!/)>", "&gt;", re.sub("<\s*(?!output)", "&lt;", value))
+        ))
+    except XMLSyntaxError:
+        # if something went horribly wrong just don't bother with escaping
+        element = Element('value')
+        element.text = value
+        return element
```

Comment:
Seems reasonable to me - it'll end up escaping everything, so the form will have undesired HTML entities in it, but the only alternative I can think of is to blank out the value, which seems worse (since you'd lose any indication of what the value used to be).

Ideally it'd fail and report something useful back (and be able to handle multiple errors in the same file) but that's probably a longer-term issue.